pkg/services/store/object/object.proto

```diff
@@ -0,0 +1,242 @@
+syntax = "proto3";
+package object;
+
+option go_package = "./;object";
+
+// Will be replaced with something from the SDK
+message UserInfo {
+  int64 id = 1; // internal grafana ID
+  string login = 2; // string ID? 
+}
+
+// Raw saved object metadata
```

Comment:
The comment says `metadata` but the object has `body` as one of its properties

Reply:
updated comment